### ISSUE TITLE
What changes should I make to the "Optimized Battery Charging" setting? 

### ISSUE THREAD

**Issue by seamusdemora** [actuallymentor/battery#225]
**What is the issue? (required)**
This isn't a "bug" per se - or maybe a bug in the documentation - or maybe I've just been unable to find it??

**What exactly did you do to produce the issue? (required)**
I've not actually installed the app yet; this is a "pre-install" Question

**Expected behavior (required)**
I would like to know how to set my "Optimized Battery Charging" option to allow `battery` to operate effectively. I am **guessing** that it should be turned OFF/Disabled, but cannot find anything in the docs to confirm that. 

Maybe this should be addressed in the README.md file???

**Screenshots (optional)**

**"Optimized Battery Charging" - ENABLED --OR-- DISABLED ?**

<img width="827" alt="Screenshot 2024-01-18 at 5 07 22 PM" src="https://github.com/actuallymentor/battery/assets/14911811/81ec5db3-cb4a-4aed-bbb2-19cdf3a1c035">

**Error logs**
N/A

Did `battery` fail to install? Use `tail -n 1000 ~/.battery/*/log` instead.
N/A

**Additional context (optional)**

**Comment by cjosue15** [comment]
I've been looking for that in the documentation, but haven't found anything.

At least in Aldente it is necessary to disable the "optimized loader" for the application to work properly.

**Comment by seamusdemora** [comment]
Aw, c'mon... this should be a simple question for anyone associated with development of this project... It's been 3 weeks now, and all I hear is "crickets".

**Comment by aschiletus** [comment]
That is the underlying idea. We are limiting with this application, we don't need anything different.

**Comment by seamusdemora** [comment]
> That is the underlying idea. We are limiting with this application, we don't need anything different.

?? What does that mean... could you please put that in plain English?

**Comment by aschiletus** [comment]
> @aschiletus
> 
> > That is the underlying idea. We are limiting with this application, we don't need anything different.
> 
> ?? What does that mean... could you please put that in plain English?

What changes should I make to the "Optimized Battery Charging" setting?

You should keep it off.

**Comment by seamusdemora** [comment]
You know...  this project is a waste of time and effort. It's going nowhere, but apparently no one has grasped that as yet.

Outta' here!

**Comment by aschiletus** [comment]
> You know... this project is a waste of time and effort. It's going nowhere, but apparently no one has grasped that as yet.
> 
> Outta' here!

Sorry for my bad English. I think we misunderstood something. The purpose of this project is simple and it does it in the best way. There is no need for constant updates because its purpose is clear and simple.

**Comment by 54huanan** [comment]
There is no need to enable Optimized Battery Charging anymore, as this app can do better.
